src/main/scala/pl/multiplex/models/MovieCategory.java

```diff
@@ -0,0 +1,13 @@
+package pl.multiplex.models;
+
+public enum MovieCategory {
```

Comment:
nie ma więcej kategorii? jak się pojawi nowa to będzie trzeba zmieniać kod

Reply:
Czyli kategorie powinienem trzymać w bazie żeby dać możliwość ich dodawania? 